src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlInternalConnectionTds.cs

```diff
@@ -2811,6 +2816,24 @@ internal void OnFeatureExtAck(int featureId, byte[] data)
                         break;
                     }
 
+                case TdsEnums.FEATUREEXT_JSONSUPPORT:
+                    {
+                        SqlClientEventSource.Log.TryAdvancedTraceEvent("<sc.SqlInternalConnectionTds.OnFeatureExtAck|ADV> {0}, Received feature extension acknowledgement for JSONSUPPORT", ObjectID);
+                        if (data.Length != 1)
+                        {
+                            SqlClientEventSource.Log.TryTraceEvent("<sc.SqlInternalConnectionTds.OnFeatureExtAck|ERR> {0}, Unknown token for JSONSUPPORT", ObjectID);
+                            throw SQL.ParsingError(ParsingErrorState.CorruptedTdsStream);
+                        }
+                        byte jsonSupportVersion = data[0];            
+                        if (jsonSupportVersion == 0 || jsonSupportVersion > 1)
```

Comment:
Recommend pulling the 1 out into a static field that better describes this version number comparison. Something like MaxSupportedJsonVersion.
```suggestion
                        if (jsonSupportVersion == 0 || jsonSupportVersion > MaxSupportedJsonVersion)
```

Reply:
done